### PR TITLE
1.2.2.0

### PR DESCRIPTION
**Pull request by skymen**
Sorry about the indent changes, it makes the diff very hard to read.

TLDR:
- Added meshPolyRenderer.js that draws the mesh polygon (in the same way that C3 draws its own mesh poly)
- Added properties for mesh poly
- Fixed bug introduced by C3's module script changes. Basically, scripts are no longer executed in the global scope and so variables need to be explicitely taken or put on the global scope through the "self" keyword, namely all your classes and the "C3" object.
- Another change I made was by passing C3 to the external scripts as a constructor param because it seems like C3 isn't defined on these script's global scope.
- Bumped version number to 1.2.2.0

### CODE REVIEW

**Comment by Straskal** [comment]
@skymen Thanks for the PR! I'll check this out after work today

I don't have time to test this on my own. So if it's good on your end, then that's good enough for me :+1:.

Also, I'd like to add you as a contributor in the addon page if you're cool with that, what is your construct username so I can @ you?

**Review by Straskal — APPROVED** [review]
Normally I'd be a bit more nit picky about the formatting lol but seeing that I haven't been maintaining this, it's all good.

**Comment by Straskal** [comment]
This PR has been merged. I bunded the addon and published it to the addon store. https://www.construct.net/en/make-games/addons/370/renderbug/versions

Would you be willing to verify that everything is in working order? Thanks again

**Comment by skymen** [comment]
> Also, I'd like to add you as a contributor in the addon page if you're cool with that, what is your construct username so I can @ you?

Yeah sure, my Construct username is @skymen :)
I'll do a round of testing. I've been using this in a project of mine so I'll see pretty quickly if something broke 👍

**Comment by skymen** [comment]
Did a few quick tests and couldn't find any issues with it so it seems all good to me
